README.md

```diff
@@ -284,8 +284,8 @@ following conditions are met:
 1. The `ruff` executable is at least version `0.5.3`
 2. The `ruff.nativeServer` setting is set to `auto` (default)
 3. None of the settings that are specific to the Python-based language server
-   are enabled. These settings include those that are marked with `[1]` in the
-   [Settings](#settings) section.
+   are enabled, as described in the [_Settings_](https://docs.astral.sh/ruff/editors/settings/#formatargs)
```

Comment:
I think we might just want to provide link to https://docs.astral.sh/ruff/editors/settings/#vs-code-specific (VS Code specific settings) and point them to look for settings that have "This setting is not used by the native language server."